### PR TITLE
Hindi translation of comdlg32.

### PR DESCRIPTION
**Pull request by arnavbhatt288**
## Purpose
Hindi translation of comdlg32.

## Proposed changes
- Added cdlg_Hi.rc
- Defined cdlg_Hi.rc in rsrc.rc

### CODE REVIEW

**Comment by HBelusca** [comment]
Hi! Thanks for your contribution.
You should however try to fix the encountered compilation errors:
```
[00:06:36] c:\reactos-cov\dll\win32\comdlg32\lang/cdlg_Hi.rc(26) : error RC2151 : cannot reuse string constants, 114(0x72) - "File not found" already defined. Cannot add 114(0x72) - "
[00:06:36] 
[00:06:36] 
[00:06:36] c:\reactos-cov\dll\win32\comdlg32\lang/cdlg_Hi.rc(27) : error RC2151 : cannot reuse string constants, 115(0x73) - "Please verify that the correct file name was given" already defined. Cannot add 115(0x73) - "
[00:06:36] 
[00:06:36] 
[00:06:36] c:\reactos-cov\dll\win32\comdlg32\lang/cdlg_Hi.rc(28) : error RC2151 : cannot reuse string constants, 116(0x74) - "File does not exist.
[00:06:36] 
[00:06:36] 
[00:06:36] Do you want to create file?" already defined. Cannot add 116(0x74) - "
[00:06:36] 
[00:06:36] 
[00:06:36] c:\reactos-cov\dll\win32\comdlg32\lang/cdlg_Hi.rc(30) : error RC2151 : cannot reuse string constants, 119(0x77) - "File already exists.
[00:06:36] 
[00:06:36] 
[00:06:36] Do you want to replace it?" already defined. Cannot add 119(0x77) - "
[00:06:36] 
[00:06:36] 
[00:06:36] c:\reactos-cov\dll\win32\comdlg32\lang/cdlg_Hi.rc(32) : error RC2151 : cannot reuse string constants, 120(0x78) - "Invalid character(s) in path" already defined. Cannot add 120(0x78) - "
[00:06:36] 
[00:06:36] 
[00:06:36] c:\reactos-cov\dll\win32\comdlg32\lang/cdlg_Hi.rc(33) : error RC2151 : cannot reuse string constants, 121(0x79) - "A filename cannot contain any of the following characters:
[00:06:36] 
[00:06:36] 
[00:06:36]                           / : < > |" already defined. Cannot add 121(0x79) - "
[00:06:36] 
[00:06:36] 
[00:06:36] c:\reactos-cov\dll\win32\comdlg32\lang/cdlg_Hi.rc(35) : error RC2151 : cannot reuse string constants, 122(0x7A) - "Path does not exist" already defined. Cannot add 122(0x7A) - "
[00:06:36] 
[00:06:36] 
[00:06:36] c:\reactos-cov\dll\win32\comdlg32\lang/cdlg_Hi.rc(36) : error RC2151 : cannot reuse string constants, 123(0x7B) - "File does not exist" already defined. Cannot add 123(0x7B) - "
[00:06:36] 
[00:06:36] 
[00:06:36] c:\reactos-cov\dll\win32\comdlg32\lang/cdlg_Hi.rc(37) : error RC2151 : cannot reuse string constants, 124(0x7C) - "The selection contains a non-folder object" already defined. Cannot add 124(0x7C) - "
[00:06:36] 
[00:06:36] 
[00:06:36] c:\reactos-cov\dll\win32\comdlg32\lang/cdlg_Hi.rc(42) : error RC2151 : cannot reuse string constants, 150(0x96) - "Up One Level" already defined. Cannot add 150(0x96) - "
[00:06:36] 
[00:06:36] 
[00:06:36] c:\reactos-cov\dll\win32\comdlg32\lang/cdlg_Hi.rc(43) : error RC2151 : cannot reuse string constants, 151(0x97) - "Create New Folder" already defined. Cannot add 151(0x97) - "
[00:06:36] 
[00:06:36] 
[00:06:36] c:\reactos-cov\dll\win32\comdlg32\lang/cdlg_Hi.rc(44) : error RC2151 : cannot reuse string constants, 152(0x98) - "List" already defined. Cannot add 152(0x98) - "
[00:06:36] 
[00:06:36] 
[00:06:36] c:\reactos-cov\dll\win32\comdlg32\lang/cdlg_Hi.rc(45) : error RC2151 : cannot reuse string constants, 153(0x99) - "Details" already defined. Cannot add 153(0x99) - "
[00:06:36] 
[00:06:36] 
[00:06:36] c:\reactos-cov\dll\win32\comdlg32\lang/cdlg_Hi.rc(46) : error RC2151 : cannot reuse string constants, 154(0x9A) - "Browse to Desktop" already defined. Cannot add 154(0x9A) - "
[00:06:36] 
[00:06:36] 
[00:06:36] c:\reactos-cov\dll\win32\comdlg32\lang/cdlg_Hi.rc(51) : error RC2151 : cannot reuse string constants, 7000(0x1B58) - "Print" already defined. Cannot add 7000(0x1B58) - "
[00:06:36] 
[00:06:36] 
[00:06:36] c:\reactos-cov\dll\win32\comdlg32\lang/cdlg_Hi.rc(53) : error RC2151 : cannot reuse string constants, 1104(0x450) - "Unreadable Entry" already defined. Cannot add 1104(0x450) - "
[00:06:36] 
[00:06:36] 
[00:06:36] c:\reactos-cov\dll\win32\comdlg32\lang/cdlg_Hi.rc(54) : error RC2151 : cannot reuse string constants, 1105(0x451) - "This value does not lie within the page range.
[00:06:36] 
[00:06:36] 
[00:06:36] Please enter a value between %1!d! and %2!d!." already defined. Cannot add 1105(0x451) - "
[00:06:36] 
[00:06:36] 
[00:06:36] c:\reactos-cov\dll\win32\comdlg32\lang/cdlg_Hi.rc(56) : error RC2151 : cannot reuse string constants, 1106(0x452) - "The 'from' entry cannot exceed the 'to' entry." already defined. Cannot add 1106(0x452) - "'
[00:06:36] 
[00:06:36] 
[00:06:36] c:\reactos-cov\dll\win32\comdlg32\lang/cdlg_Hi.rc(57) : error RC2151 : cannot reuse string constants, 1107(0x453) - "Margins overlap or fall outside Paper boundaries.
[00:06:36] 
[00:06:36] 
[00:06:36] Please reenter margins." already defined. Cannot add 1107(0x453) - "
[00:06:36] 
[00:06:36] 
[00:06:36] c:\reactos-cov\dll\win32\comdlg32\lang/cdlg_Hi.rc(59) : error RC2151 : cannot reuse string constants, 1108(0x454) - "The 'Number of copies' field cannot be empty." already defined. Cannot add 1108(0x454) - "'
[00:06:36] 
[00:06:36] 
[00:06:36] c:\reactos-cov\dll\win32\comdlg32\lang/cdlg_Hi.rc(60) : error RC2151 : cannot reuse string constants, 1109(0x455) - "This large number of copies is not supported by your printer.
[00:06:36] 
[00:06:36] 
[00:06:36] Please enter a value between 1 and %d." already defined. Cannot add 1109(0x455) - "
[00:06:36] 
[00:06:36] 
[00:06:36] c:\reactos-cov\dll\win32\comdlg32\lang/cdlg_Hi.rc(62) : error RC2151 : cannot reuse string constants, 1110(0x456) - "A printer error occurred." already defined. Cannot add 1110(0x456) - "
[00:06:36] 
[00:06:36] 
[00:06:36] c:\reactos-cov\dll\win32\comdlg32\lang/cdlg_Hi.rc(63) : error RC2151 : cannot reuse string constants, 1111(0x457) - "No default printer defined." already defined. Cannot add 1111(0x457) - "
[00:06:36] 
[00:06:36] 
[00:06:36] c:\reactos-cov\dll\win32\comdlg32\lang/cdlg_Hi.rc(64) : error RC2151 : cannot reuse string constants, 1112(0x458) - "Cannot find the printer." already defined. Cannot add 1112(0x458) - "
[00:06:36] 
[00:06:36] 
[00:06:36] c:\reactos-cov\dll\win32\comdlg32\lang/cdlg_Hi.rc(65) : error RC2151 : cannot reuse string constants, 1113(0x459) - "Out of memory." already defined. Cannot add 1113(0x459) - "
[00:06:36] 
[00:06:36] 
[00:06:36] c:\reactos-cov\dll\win32\comdlg32\lang/cdlg_Hi.rc(66) : error RC2151 : cannot reuse string constants, 1114(0x45A) - "An error occurred." already defined. Cannot add 1114(0x45A) - "
[00:06:36] 
```

**Comment by arnavbhatt288** [comment]
@arnavbhatt288 I don't really know what is the issue so can you explain me please?

File: dll/win32/comdlg32/lang/cdlg_Hi.rc [hunk 1]
```diff
@@ -0,0 +1,522 @@
...
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
+ */
+
+LANGUAGE LANG_ENGLISH, SUBLANG_DEFAULT
```
Comment (HBelusca) [inline dll/win32/comdlg32/lang/cdlg_Hi.rc]: Please change this to the correct values for hindi.

**Comment by HBelusca** [comment]
When the resource compiler tells that there are duplicated/re-used identifiers for the translated strings (as shown above), one of the common errors, which was actually what happened there, is that the translation resource file uses the very same language identifier as another existing one within the same project, namely in this case the english one.
This is what you should fix in your case.

File: dll/win32/comdlg32/lang/cdlg_Hi.rc (hunk 1 above)
Comment (arnavbhatt288) [reply dll/win32/comdlg32/lang/cdlg_Hi.rc]: oopsies, forgot about this one.

**Comment by arnavbhatt288** [comment]
@HBelusca I completely forgot about it. I will take care in future. Thanks!

**Pull request merged**